src/WooCommerce.php

```diff
@@ -1509,8 +1511,46 @@ private static function getRepricingOptionsDropdownValues(\WC_Product $product)
       !empty($brand) ? $brand . ' | no repricing' : 'no repricing' => __('No Repricing', Plugin::L10N),
       !empty($brand) ? $brand . ' | lower prices only' : 'lower prices only' => __('Lower prices only', Plugin::L10N),
     ];
-    
+
     return $dropdown_values;
   }
 
+  /**
+   * Separates the shipping methods in new lines in emails.
+   *
+   * @param  array $total_rows
+   * @param \WC_Order $order
+   * @return array
+   */
+  public static function woocommerce_get_order_item_totals($total_rows, \WC_Order $order, $tax_display):array {
+    $shipping_methods = $order->get_shipping_methods();
+    $currency = ' ' . get_woocommerce_currency_symbol();
+
+    if(count($shipping_methods) > 1) {
+
+      $shipping_methods_row = '';
+      $count = 1;
+
+      foreach($shipping_methods as $shipping_method) {
+        // Calculate => Shipping Price + Tax + Symbol
+        $shipping_price = floatval($shipping_method->get_total());
+        $shipping_price_tax = floatval($shipping_method->get_data()['total_tax']);
+        $shipping_price_total = $shipping_price + $shipping_price_tax;
+        $shipping_price_total_with_symbol = $shipping_price_total . $currency;
```

Comment:
```suggestion
        // Calculate => Shipping Price + Tax + Symbol
        $shipping_price = floatval($shipping_method->get_total());
        $shipping_price_tax = floatval($shipping_method->get_data()['total_tax']);
        $shipping_price_total = $shipping_price + $shipping_price_tax;
        $shipping_price_total_with_symbol = $shipping_price_total . $currency;
```

Let's create a new method that takes care of this following the single responsibility principle.